docs/localization/i-cant-install-the-headless-cms-plugins-in-my-account.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "I can’t install the Headless CMS plugins in my account"
+excerpt: "Windows users may encounter the 'EPERM: operation not permitted' error during the installation of the Headless CMS plugin."
```

Comment:
Como agora são dois problemas relacionados a instalação, talvez deixar um pouco mais genérico aqui. Algo como "Users may encounter errors related to symbolic links and node version incompatibility."